azure-go-sdk-conceptual/azure-sdk-go-auth-aad-samples.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure SDK for Go samples for authentication and AAD 
+description: Selected samples for working with Azure Active Directory (AAD) and authentication from the Azure SDK for Go.
+author: sptramer
+ms.author: sttramer
+ms.date: 03/21/2018
+ms.topic: article
+ms.devlang: go
+manager: carmonm
+---
+
+# Azure SDK for Go samples for authentication and AAD
+
+The following table links to selected samples of Go source code that you can use to authenticate with the Azure SDK for Go and work with Azure Activce Directory (AAD) services. To get all of the sample code and tests for authentication and AAD, use the `go get` command:
+
```

Comment:
I think `git clone` would be better here, cause we don't intend for users to link in the sample code in apps. In fact our goal is mainly to give them something to read and copy, so perhaps we don't need any hints on how to download?